lib/utils/path-helpers.js

```diff
@@ -17,6 +17,10 @@ const pathHelpers = {
     return moduleName.endsWith('/') ? ''
       : moduleName.replace(path.dirname(moduleName), '').replace('/', '');
   },
+  getFileDir: (filePath) => {
+    const PATH_SLASH = process.platform === 'win32' ? '\\' : '/';
+    return filePath.substring(0, filePath.lastIndexOf(PATH_SLASH));
```

Comment:
What's the difference between this custom impl and `path.dirname`?

Reply:
good point. Not sure of a reasoning behind that.

Was a code carried from the original setup. Might have to do with windows vs linux? I'll look into that later for future refactoring.

Reply:
As far as I remember, the built-in 'path' module irons out the differences in path representations across platforms, that's why I asked.